.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 120
+application_import_names = spellchecker
```

Comment:
Could you add `import-order-style = pycharm` as well to stay consistent with our other projects?